packages/api/src/module/write/email-confirmation/presentation/rest/email-confirmation.rest-controller.ts

```diff
@@ -21,11 +30,33 @@ export class EmailConfirmationRestController {
   @HttpCode(204)
   async emailConfirmation(
     @JwtUserId() userId: UserId,
-    @Body() { confirmationFor }: EmailConfirmationBody,
+    @Body() { confirmationFor }: RequestEmailConfirmationBody,
   ): Promise<void> {
     const command = this.commandFactory.applicationCommand((idGenerator) => ({
       class: RequestEmailConfirmationApplicationCommand,
-      ...requestEmailConfirmationCommand({ userId, confirmationToken: idGenerator.generate(), confirmationFor }),
+      ...requestEmailConfirmationCommand({
+        userId,
+        confirmationToken: idGenerator.generate(),
+        confirmationFor,
+      }),
+    }));
+
+    await this.commandBus.execute(command);
+  }
+
+  @Post(APPROVE_ENDPOINT)
+  @HttpCode(204)
+  async approveEmailConfirmation(
```

Comment:
Could you add OpenAPI documentation (for Swagger UI, yaml is /packages/api/rest-api-docs.yaml file - you can find it under /swagger endpoint while running the app) and tests validating contract for his controller? 
Lastly we've introduced this testing strategy. You can find example of tests in file: 
`packages/api/src/module/write/user-registration/presentation/rest/user-registration.rest-controller.spec.ts`

Reply:
Will be done in next issue